web/src/components/layout/layout.tsx

```diff
@@ -162,6 +166,17 @@ class Layout extends React.PureComponent<LayoutProps, LayoutState> {
 
     return (
       <div id="main" className={className}>
+        {showWelcomeModal && (
+          <WelcomeModal
+            onRequestClose={() => {
+              this.setState({ showWelcomeModal: false });
+            }}
+            onClick={() => {
+              window.location.pathname = '/login';
+            }}
+            team="SAP"
```

Comment:
Note: eventually this will be determined by query strings, but for now I just hardcoded it.